assets/js/components/Table/EmptyTable.stories.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import EmptyTable from './EmptyTable';
+
+export default {
+  title: 'EmptyTable',
+  component: EmptyTable,
+};
+
+export function Default(args) {
+  return <EmptyTable {...args} />;
```

Comment:
Instead of having a story for `EmptyTable`, you could have a story for the `Table` component rendering the empty state :+1:

Reply:
I deleted the EmptyTable story as the Table component already has it covered!